specifyweb/businessrules/views.py

```diff
@@ -1 +1,284 @@
+                            "additionalProperties": {
+                                "type": "array",
+                                "description": "The array of uniqueness rules for a given table",
+                                "items": UniquenessRuleSchema
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "post": {
+        "requestBody": {
+            "required": True,
+            "content": {
+                "application/json": {
+                    "schema": {
+                        "type": "object",
+                        "properties": {
+                            "rules": {
+                                "type": "array",
+                                "description": "The array of uniqueness rules for a given table",
+                                "items": UniquenessRuleSchema
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "responses": {
+            "201": {
+                "description": "Uniqueness rules properly updated and/or created",
+                "content": {
+                    "application/json": {
+                        "schema": {
+                            "type": "object"
+                        }
+                    }
+                }
+            }
+        }
+    }
+})
+@login_maybe_required
+@require_http_methods(['GET', 'POST'])
+def uniqueness_rule(request, discipline_id):
+    data = {}
+    DB_RESOURCE_URI = "/_database"
+
+    try:
+        model = request.GET["model"]
+    except:
+        model = None
+
+    if request.method == 'GET':
+        rules = UniquenessRule.objects.filter(discipline=discipline_id)
+        for rule in rules:
+            rule_fields = rule.splocalecontaineritems.filter(
+                uniquenessrule_splocalecontaineritem__isScope=0)
+            _scope = rule.splocalecontaineritems.filter(
+                uniquenessrule_splocalecontaineritem__isScope=1)
+            scope = None if len(_scope) == 0 else _scope
+
+            table = rule_fields[0].container.name
+            if model is not None and table.lower() != model.lower():
+                continue
+            if table not in data.keys():
+                data[table] = []
+            data[table].append({"id": rule.id, "fields": [obj_to_data(field) for field in rule_fields], "scope": obj_to_data(
+                scope[0]) if scope is not None else None, "isDatabaseConstraint": rule.isDatabaseConstraint})
+
+    elif request.method == 'POST':
+        rules = json.loads(request.body)['rules']
+        discipline = models.Discipline.objects.get(id=discipline_id)
+        for rule in rules:
+            fetched_scope = None if rule["scope"]["resource_uri"] == DB_RESOURCE_URI else models.Splocalecontaineritem.objects.get(
+                id=rule["scope"]["id"])
```

Comment:
DB Resource URI removed in d9c8caabd37a67d3d3bd0606c25186366aaecd61. 

However, it will be refactored to instead send an empty array if there is no scope.